.github/workflows/dep-diff-pull_request.yml

```diff
@@ -8,7 +8,7 @@ on:
     - master
 env:
   # The modules to check for dependencies. If there is more than one they are comma separated
-  MODULES_TO_CHECK: core-feature-pack/common
+  MODULES_TO_CHECK: core-feature-pack/common,testbom
```

Comment:
After checking the workflow files, added `testbom` does not make much sense because:
* It contains `test` only dependencies
* It contains only `<dependencyManagement>`, so no dependency:list output to compare at all

As for the CI failure, I think we can ignore it, because this PR introduces a new artifact, but the step of `Build PR` tries to build the modules in `MODULES_TO_CHECK` only. I don't see a good way to solve it. :)

@bstansberry if you agree, I will reset this workflow file and trigger another CI run. :)